From e12e188f393a3f2a867f7ea4abfb5d2b5fe53916 Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Wed, 17 Oct 2018 15:56:46 +0200
Subject: [PATCH 0715/1051] Revert "crypto: inside-secure: adjust firmware
 path"

This reverts commit 771e451d775cf5cb6ecc05db75b19964722afbac.

Change-Id: I974ec3866e68e4deac579145212886ed4c3334c3
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/60354
Reviewed-by: Igal Liberman <igall@marvell.com>
Tested-by: Igal Liberman <igall@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/crypto/inside-secure/safexcel.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/crypto/inside-secure/safexcel.c b/drivers/crypto/inside-secure/safexcel.c
index 33d268dd98f2..552188689c6b 100644
--- a/drivers/crypto/inside-secure/safexcel.c
+++ b/drivers/crypto/inside-secure/safexcel.c
@@ -178,8 +178,7 @@ static void eip197_write_firmware(struct safexcel_crypto_priv *priv,
 
 static int eip197_load_firmwares(struct safexcel_crypto_priv *priv)
 {
-	static const char * const fw_name[] = {"eip197/197b/ifpp.bin",
-					       "eip197/197b/ipue.bin"};
+	const char *fw_name[] = {"ifpp.bin", "ipue.bin"};
 	const struct firmware *fw[FW_NB];
 	char fw_path[31], *dir = NULL;
 	int i, j, ret = 0, pe;
-- 
2.17.1

